lib/kafo/configuration.rb

```diff
@@ -240,6 +240,23 @@ def answers
       @data
     end
 
+    def run_migrations
+      migrations = Kafo::Migrations.new(migrations_dir)
+      migrations.load_migrations
```

Comment:
no I meant moving `Kafo::Migrations#load_migrations` to become part of `Kafo::Migrations#initialize`, so it wouldn't have to be called explicitly here, I don't see any use of `Kafo::Migrations` that wouldn't have loaded migration already